src/SIL.XForge.Scripture/ClientApp/e2e/pre_merge_ci.sh

```
@@ -0,0 +1,90 @@
+  kill -KILL "${pid}" 2>/dev/null
+}
+
+function startServer() {
+  cd "${SCRIPT_DIR}/../.."
+  local DOTNET_LOG
+  DOTNET_LOG="$(mktemp)"
+  output "Logging dotnet output to ${DOTNET_LOG}"
+  export ASPNETCORE_ENVIRONMENT="Development"
+  nohup dotnet run &>"${DOTNET_LOG}" &
+  local SERVER_PID="$!"
+  trap "shutDownServer ${SERVER_PID}" EXIT
```

Shellcheck (0.10.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)